scripting/get5.sp

```diff
@@ -260,6 +261,8 @@ public void OnPluginStart() {
                    "If set to 0, get5 will not force players to the correct team based on steamid");
   g_DemoNameFormatCvar = CreateConVar("get5_demo_name_format", "{MATCHID}_map{MAPNUMBER}_{MAPNAME}",
                                       "Format for demo file names, use \"\" to disable");
+  g_DisplayGotvVeto = CreateConVar("get5_display_gotv_veto", "0",
+                                      "Whether to wait for map vetos to be printed to GOTV before changing map, use \"\" to disable");
```

Comment:
Remove the 
>, use \"\" to disable

part. It's misleading since the default is 0 already, and there's lots of other bool cvars that control 'whether' something happens, and how to disable it isn't part of the description.